src/components/Select/Select.jsx

```diff
@@ -4,14 +4,13 @@ import * as Styled from './styles';
 import Arrow from '../Icons/Arrow';
 import { useClickOutside } from '../../hooks/hooks';
 
-const Select = ({ label, placeholder, defaultValue, options }) => {
-  const [activeOption, setActiveOption] = useState(defaultValue);
+const Select = ({ label, placeholder, options, chooseOption, active }) => {
```

Comment:
Prefer to use `onChange` callback name instead of `chooseOption`. All component events/callbacks should be prefixed with `on`

Reply:
You can rename this prop and merge PR by yourself